src/style.css

```diff
@@ -0,0 +1,113 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px ;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 0 0;
+}
+
+.code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 16px;
+  margin-top: 16px;
```

Comment:
no external styles on blocks

Reply:
@Sardorbek1122 you don't seem to fix `external styles on blocks`
you could just combine block styles with external styles like so:
`<div class="stars stars-external">`

and list all external styling in 
```
.stars-external {
   ...
}
```